diff --git a/index.js b/index.js
index 609cf3113940d6811fd9798e27e483e460ff6761..1e8e94e9a52a782e2451c42627fc25bbcef95334 100644
--- a/index.js
+++ b/index.js
@@ -15,7 +15,7 @@ var customInspectSymbol =
     ? Symbol['for']('nodejs.util.inspect.custom') // eslint-disable-line dot-notation
     : null
 
-exports.Buffer = Buffer
+globalThis.Buffer = exports.Buffer = Buffer
 exports.SlowBuffer = SlowBuffer
 exports.INSPECT_MAX_BYTES = 50
 
